src/disc/Albany_AbstractMeshFieldAccessor.hpp

```diff
@@ -54,6 +54,9 @@ class AbstractMeshFieldAccessor
   virtual void fillSolnMultiVector (Thyra_MultiVector&   soln,
                                     const dof_mgr_ptr_t& sol_dof_mgr,
                                     const bool           overlapped) = 0;
+  virtual void fillSolnSensitivity (Thyra_MultiVector&   dxdp,
```

Comment:
This, and all the dxdp changes in the PR, are so that we can run a pb that computes sensitivities and do adaptation. I haven't tested the dxdp functionality though, so more work may be needed down the road.